tests/integration/ceph_base_object_test.go

```diff
@@ -53,22 +53,24 @@ var (
 	region                 = "us-east-1"
 	maxObject              = "2"
 	bucketStorageClassName = "rook-smoke-delete-bucket"
+	j                      = 0
```

Comment:
```suggestion
	j                      = 0
```
??

Reply:
Just a `retry count` variable for testing.